modules/php/States/EngineTrait.php

```diff
@@ -133,7 +133,6 @@ function actAnytimeAction($choiceId, $auto = false)
   #[CheckAction(false)]
   function actTakeAtomicAction(string $actionName, #[JsonParam] array $actionArgs)
   {
-    self::checkAction($actionName);
```

Comment:
Why do you remove this ?

Reply:
#[CheckAction(false)] is doing this now, we don't need it, do we?

Reply:
Reverted this

Reply:
We need this because the js action is actually different from the actionName registred in states.php so that's the part where we want to check the action name is legit

Reply:
Ok, sure, thanks for the explanation!